output/report.go

```diff
@@ -29,29 +29,23 @@ import (
 
 // Reporter is responsible for outputting test results
 type Reporter interface {
-	Stream(config *plan.Config, tests <-chan execute.TestResponse) Summary
-}
-
-// ReporterFunc streams test results to the console
-type ReporterFunc func(config *plan.Config, tests <-chan execute.TestResponse) Summary
-
-// Stream allows ReporterFunc to fulfill the Reporter interface
-func (f ReporterFunc) Stream(config *plan.Config, tests <-chan execute.TestResponse) Summary {
-	return f(config, tests)
+	Start(config *plan.Config) error
```

Comment:
It does depend on the size of the config and how often you'll be calling the method. If you only ever call the method once for the whole process, then it's probably not worth optimizing. If this is something you'll call often, then it's probably worth using pointers.

The Go CodeReviewComments page has some info on this too:
https://github.com/golang/go/wiki/CodeReviewComments#receiver-type

```
If the receiver is a large struct or array, a pointer receiver is more efficient. How large is large? Assume it's equivalent to passing all its elements as arguments to the method. If that feels too large, it's also too large for the receiver.
```